.github/workflows/build-and-test.yml

```diff
@@ -81,3 +81,18 @@ jobs:
 
       - name: E2E Node Federation
         run: npx nx run-many --target=serve --projects=node-local-remote,node-remote,node-dynamic-remote-new-version,node-dynamic-remote --parallel=10 & echo "done" && sleep 25 && npx nx run-many --target=serve --projects=node-host & sleep 5 && npx wait-on tcp:3333 && npx nx run node-host-e2e:test:e2e
+
+      - name: E2E Test for Modern.js Dev
```

Comment:
Split the test out and add whether the change affects it to decide whether to run it